### PR TITLE
Bug fix: handle zip content type and remove use of undefined variable

### PR DESCRIPTION
**Pull request by rakeshsahuu**
Two changes,
1. Updated logic to skip warning message for zip content type, when "get logs component=controller" is used to get the logfile.zip
2. Removed use of the undefined variable, this ensures "http push <firmware_file> /redfish/v1/UpdateService/FWUpdate { "Targets": [], "@Redfish.OperationApplyTime": "Immediate" }" does not exit with an error.

### CODE REVIEW

File: commands/systems/http_push.py [hunk 1]
```diff
@@ -86,8 +86,6 @@ def display_results(self, redfishConfig):
             Trace.log(TraceLevel.INFO, '[] Status       : {}'.format(self.link.urlStatus))
             Trace.log(TraceLevel.INFO, '[] Reason       : {}'.format(self.link.urlReason))
 
-        if self.context != '':
-            print(' [] Context    : {}'.format(self.context))
```
Comment (jskazinski) [inline commands/systems/http_push.py:90]: Please make a small change to use: Trace.log(TraceLevel.INFO, ' [] Context : {}'.format(self.context))

**Review by jskazinski — CHANGES_REQUESTED** [review]
Changes look to me, except please use Trace.log instead of print so we can adjust the level of output.

**Review by jskazinski — CHANGES_REQUESTED** [review]
Please also make the following changes:
- Bump the version to 2.2.5 in version.py
- Add an entry in CHANGELOG.md reflecting this version and changes.
-

File: commands/systems/http_push.py (hunk 1 above)
Comment (rakeshsahuu) [reply commands/systems/http_push.py:90]: Hi Joe, there is no variable named "context" in CommandHandler class, so isn't it worth removing it?

OR,
I can also do,
if self.context:
            Trace.log(TraceLevel.INFO, ' [] Context : {}'.format(self.context))

Comment (jskazinski) [reply commands/systems/http_push.py:90]: Yes, please do your OR statement, where you are using Trace.log instead of print. Thanks.

**Comment by jskazinski** [comment]
Thanks for updating the version and CHANGELOG.

**Pull request merged**